### ISSUE TITLE
Link to mobile app stores in the 'Free app' blurb

### ISSUE THREAD

**Issue by Phrohdoh** [supergreenlab/supergreenlab.com#8]

**Comment by vitaminwater** [comment]
Yeah I need to finish shooting the different versions here: https://github.com/supergreenlab/SuperGreenApp/releases

I stopped yesterday to download a windows 10 vm for the windows version, I should be able to finish that today
